checksumpackage

```diff
@@ -114,6 +106,8 @@ _version_dfxml(){
 }
 
 while [ "${*}" != "" ] ; do
+    ABS_PATH=$(cd "$1"; pwd)
+    INPUT=$(basename "$1")
     _log -b
     PACKAGE="${ABS_PATH}"
     if [ -d "${PACKAGE}" ] ; then
```

Comment:
On examination, that check was indeed insufficient!!  Fixed with https://github.com/mediamicroservices/mm/pull/84/commits/2e422689e0d262196adac488f2add13ce0d09d00 - script should now break loop and exit on illegal input.